src/libsystemd-network/dhcp-lease-internal.h

```diff
@@ -38,6 +39,14 @@ struct sd_dhcp_route {
         unsigned char dst_prefixlen;
 };
 
+struct sd_dhcp_raw_option {
+        LIST_FIELDS(struct sd_dhcp_raw_option, options);
+
+        uint8_t tag;
+        uint8_t *data;
+        uint8_t length;
+};
```

Comment:
Change the order of the fields to tag, length, data to have a little less memory wasted on alignment.